packages/react-magma-dom/src/components/Pagination/SimplePagination.tsx

```diff
@@ -0,0 +1,175 @@
+import * as React from 'react';
+import { ButtonColor, ButtonShape, ButtonSize, ButtonVariant } from '../Button';
+import { I18nContext } from '../../i18n';
+import { Tooltip } from '../Tooltip';
+import { ArrowBackIcon, ArrowForwardIcon } from 'react-magma-icons';
+import styled from '../../theme/styled';
+import { ThemeContext } from '../../theme/ThemeContext';
+import { NativeSelect } from '../NativeSelect';
+import { NavButton, PaginationProps, PageButtonSize } from './';
+import { useGenerateId } from '../../utils';
+import { Spacer } from '../Spacer';
+import { VisuallyHidden } from '../VisuallyHidden';
+import { transparentize } from 'polished';
+
+function buildLabelColor(props) {
+  if (props.isInverse) {
+    if (props.disabled) {
+      return transparentize(0.8, props.theme.colors.neutral100);
+    }
+    return props.theme.colors.neutral100;
+  }
+  if (props.disabled) {
+    return props.theme.colors.neutral500;
+  }
+  return props.theme.colors.neutral700;
+}
+
+const StyledWrapper = styled.div<{
+  disabled?: boolean;
+  isInverse?: boolean;
+}>`
+  display: flex;
+  align-items: center;
+  max-width: 360px;
+  label {
+    color: ${buildLabelColor};
+    font-size: ${props => props.theme.typeScale.size02.fontSize};
+    font-weight: 500;
+  }
+`;
+
+const StyledNativeSelect = styled(NativeSelect)<{
+  disabled?: boolean;
+}>`
+  select {
+    color: ${props => (props.disabled ? props.theme.colors.neutral500 : '')};
+    cursor: ${props => (props.disabled ? 'not-allowed' : '')};
+  }
```

Comment:
Just curious why this doesn't work by default with our NativeSelects?

Reply:
All set, removed those repetitive attributes.